src/OpenTelemetry.Api/README.md

```diff
@@ -3,63 +3,62 @@
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Api.svg)](https://www.nuget.org/packages/OpenTelemetry.Api)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Api.svg)](https://www.nuget.org/packages/OpenTelemetry.Api)
 
+* [Installation](#installation)
+* [Getting started](#getting-started)
+  * [Traceing API](#tracing-api)
+  * [Metrics API](#metrics-api)
+* [Introduction to OpenTelemetry .NET Traceing
+  API](#introduction-to-opentelemetry-net-tracing-api)
+* [Instrumenting a library/application with .NET Activity
+  API](#instrumenting-a-libraryapplication-with-net-activity-api)
+  * [Basic usage](#basic-usage)
+  * [Activity creation options](#activity-creation-options)
+  * [Adding Events](#adding-events)
+  * [Setting Status](#setting-status)
+* [Instrumenting a library/application with OpenTelemetry.API
+  Shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim)
+* [References](#references)
+
 ## Installation
 
 ```shell
 dotnet add package OpenTelemetry.Api
 ```
 
-## Getting Started
-
-* [Getting Started](#getting-started)
-  * [OpenTelemetry API](#opentelemetry-api)
-    * [Tracer API](#tracer-api)
-    * [Metric API](#metric-api)
-  * [Introduction to OpenTelemetry .NET Tracer
-    API](#introduction-to-opentelemetry-net-tracer-api)
-  * [Instrumenting a library/application with .NET Activity
-    API](#instrumenting-a-libraryapplication-with-net-activity-api)
-    * [Basic usage](#basic-usage)
-    * [Activity creation options](#activity-creation-options)
-    * [Adding Events](#adding-events)
-    * [Setting Status](#setting-status)
-  * [Instrumenting a library/application with OpenTelemetry.API
-    Shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim)
-
-## OpenTelemetry API
+## Getting started
 
 Application developers and library authors use OpenTelemetry API to instrument
 their application/library. The API only surfaces necessary abstractions to
 instrument an application/library. It does not address concerns like how
 telemetry is exported to a specific telemetry backend, how to sample the
-telemetry, etc. The API consists of [Tracer
+telemetry, etc. The API consists of [Tracing
 API](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md),
-[Metric
+[Metrics
 API](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md),
 [Context and Propagation
 API](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/context),
 and a set of [semantic
 conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/trace/semantic_conventions).
 
-### Tracer API
+### Traceing API
```

Comment:
mm.. How about Trace API, Metric API? Traceing doesn't feel natural to me.

Reply:
or did you mean Tracing ?

Reply:
`Traceing` was a typo.
We need to call it `Tracing API` rather than `Tracer API` or `Trace API` according to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#tracing-api).